themes/bootstrap3/templates/Recommend/ProQuestFSGResults.phtml

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Looks like the bootstrap3 templates you added for this PR didn't get deleted when you merged in the removal of that theme. We should get rid of them now!

Reply:
D'oh!  When I merged in dev I saw all of the deletions of the existing bootstrap3 templates I had edited, but didn't think to also remove the new ones.  Fixed.